src/utils/mtev_lfu.c

```diff
@@ -236,6 +263,12 @@ touch_lfu_cache_no_lock(mtev_lfu_t *c, struct lfu_entry *e)
 mtev_boolean
 mtev_lfu_put(mtev_lfu_t *lfu, const char *key, size_t key_len, void *val)
 {
+  /* a max size of zero means to disable the LFU */
+  if (lfu->max_entries == 0) {
+    /* we don't want calling code to error */
+    return mtev_true;
```

Comment:
`mtev_lfu_get` or `remove` could always return an invalid value because something else may have come along to evict that item at any time.

What I wanted to avoid was someone writing code like:

```
if (mtev_lfu_put(lfu, mykey, mykeylen, myval) == mtev_false) {
  mtevFatal(mtev_error, "Cannot cache item in LFU\n");
}
```
And then changing the LFU size to zero in the config so they can disable it causing the app to crash.